src/ui/popup/manage-settings-page/sync-config.tsx

```diff
@@ -0,0 +1,20 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import CheckButton from '../check-button';
+
+let synced = false;
+export default function SyncConfigButton(props: ViewProps) {
+    function syncConfig() {
+        if (!synced) {
+            synced = true;
+            props.actions.loadConfig({local: false});
+        }
+    }
+    return (
+        <CheckButton
+            checked={synced}
+            label='Sync config'
```

Comment:
`config` is equal to `settings` for a usual user, rename to `Synchronize sites fixes`